packages/react-router-dom/modules/Link.js

```diff
@@ -5,6 +5,9 @@ import invariant from 'invariant'
 const isModifiedEvent = (event) =>
   !!(event.metaKey || event.altKey || event.ctrlKey || event.shiftKey)
 
+const resolveToLocation = (to, location) =>
```

Comment:
Bikeshedding, but I don't love including the word "resolve" in this function name because that seems to infer some sort of relative/absolute path resolution.